cmd/block-generator/generator/generate.go

```diff
@@ -390,7 +391,6 @@ func convert(txn transactions.Transaction, ad transactions.ApplyData) transactio
 	stxnib := transactions.SignedTxnInBlock{
 		SignedTxnWithAD: withAd,
```

Comment:
we should probably use `BlockHeader.EncodeSignedTxn()`

Reply:
Nice, thanks for the tip! Still a little work to do for Assets but at least payment txns are working.